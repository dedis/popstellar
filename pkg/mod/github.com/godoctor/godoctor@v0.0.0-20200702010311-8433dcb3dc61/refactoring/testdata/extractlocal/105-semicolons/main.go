package main

import "fmt"

func main() {
	i := 0; j := 1; k := /* seriously */ j + 1 /* dude */
		// <<<<< var,6,39,6,43,newVar,pass
	fmt.Printf("%d %d %d\n", i, j, k)
}
