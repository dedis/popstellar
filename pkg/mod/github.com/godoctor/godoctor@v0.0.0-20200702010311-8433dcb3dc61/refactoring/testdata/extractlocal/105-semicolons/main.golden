package main

import "fmt"

func main() {
	i := 0
	j := 1
	newVar := j + 1
	k := /* seriously */ newVar /* dude */
	// <<<<< var,6,39,6,43,newVar,pass
	fmt.Printf("%d %d %d\n", i, j, k)
}
